programs/bpf/c/sdk/inc/stdlib.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+#include <solana_sdk.h>
```

Comment:
Is this file a placeholder for future stdlib definitions/declarations?

Reply:
Yeah.  Until we pull in a real libc, my intent is to fill out these stub libc headers on demand as we try to crank up the complexity of our BPF experiments and run into the next level if LLVM limitations